cobspec/src/main/java/com/saralein/cobspec/controller/LogController.java

```diff
@@ -0,0 +1,32 @@
+package com.saralein.cobspec.controller;
+
+import com.saralein.server.controller.Controller;
+import com.saralein.server.request.Request;
+import com.saralein.server.response.Response;
+import com.saralein.server.response.ResponseBuilder;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class LogController implements Controller {
+    private final Path log;
+
+    public LogController(Path log) {
+        this.log = log;
+    }
+
+    public Response createResponse(Request request) {
```

Comment:
This method name has bugged for awhile, and it wasn't until I came across a snippet in a book I am reading that I figured out how to express it.

My issue with this name is that it reveals information about how it is implemented. If the method did not `new` a response object, would the name end up changing? I am not sure what I would rename it to, maybe just `response` or `respond`.